### ISSUE TITLE
False positive for SlevomatCodingStandard.Classes.PropertyDeclaration.MultipleSpacesBetweenTypeHintAndProperty

### ISSUE THREAD

**Issue by MasterOdin** [slevomat/coding-standard#1760]
This file from my project (https://github.com/cakephp/phinx/blob/be536ffb2a5e97fd36d4935964370216531f3637/tests/Phinx/Config/AbstractConfigTest.php) is suddenly failing with 4.19.0 with the following:

```
FILE: /home/mpeveler/github/phinx/tests/Phinx/Config/AbstractConfigTest.php
------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------
 14 | ERROR | [x] There must be exactly one space between type hint and property $migrationPath.
    |       |     (SlevomatCodingStandard.Classes.PropertyDeclaration.MultipleSpacesBetweenTypeHintAndProperty)
------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------------------------------

Time: 70ms; Memory: 10MB
```

If I remove the two properties or make the first one `protected string $migrationPath`, then the error goes away. Trying to run `vendor/bin/phpcbf` and it fails to fix whatever error it's complaining about. Have absolutely no idea why this singular class within my project has issues as it's structured like all my other test classes.

The file validated fine with 4.18.1 and earlier.

**Comment by pekral** [comment]
Same problem in my project.
